Nova documentação de Notificações (feature/new-notifications) (#2426)
mercadopago/devsite-docs: guides/additional-content/your-integrations/ipn.pt.md

| Alertas de fraude após o processamento de um pedido | Alertas de fraude | `delivery_cancellation` | Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------<br>Checkout PRO |
| Criação, fechamento ou expiração de pedidos. |  Ordens comerciais | `merchant_order` | Checkout Pro<br>Código QR  |
| Abertura de estornos, alterações de status e modificações relacionadas às liberações de dinheiro.   |   Estornos | `chargebacks`  | Checkout Pro<br>Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------ <br>Checkout Bricks |

> WARNING
>
> Importante
>
> Em caso de dúvidas sobre quais tópicos ativar ou quais eventos serão notificados, consulte a documentação [Informações adicionais sobre notificações](/developers/pt/docs/your-integrations/notifications/additional-info). 

4. Por fim, clique em **Salvar**. 

## Configuração durante a criação de pagamentos

Durante o processo de criação de pagamentos, é possível configurar a URL de notificação de maneira específica para cada pagamento. Para isso, utilize o campo `notification_url` e implemente o receptor de notificações necessário.  

> > WARNING

Reviewer: ```suggestion
----[mla, mlb, mlm]----
> WARNING
```

>
> Importante
>

> > Este método não permite configurar notificações para o tópico `point_integration_ipn`. Para ativá-lo, utilize as [configurações através de Suas integrações](/developers/pt/docs/your-integrations/notifications/ipn#configuraoviasuasintegraes).

Reviewer: ```suggestion
> Este método não permite configurar notificações para o tópico `point_integration_ipn`. Para ativá-lo, utilize as [configurações através de Suas integrações](/developers/pt/docs/your-integrations/notifications/ipn#configuraoviasuasintegraes).

------------
```

 
A seguir, explicamos como realizar esta configuração utilizando nossoss SDKs.

> 1. No campo `notificaction_url`, indique a URL do qual serão recebidas as notificações como exemplificado abaixo. Para receber notificações exclusivamente via IPN e não via Webhooks, adicione o parâmetro `source_news=webhooks` à `notification_url`. Por exemplo: "https://www.yourserver.com/notifications?source_news=webhooks".

Reviewer: ```suggestion
1. No campo `notificaction_url`, indique a URL do qual serão recebidas as notificações como exemplificado abaixo. Para receber notificações exclusivamente via IPN e não via Webhooks, adicione o parâmetro `source_news=webhooks` à `notification_url`. Por exemplo: `https://www.yourserver.com/notifications?source_news=webhooks`.
```

| Abertura de estornos, alterações de status e modificações relacionadas às liberações de dinheiro.   |   Estornos | `chargebacks`  | Checkout Pro<br>Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------ <br>Checkout Bricks |

> WARNING
>
> Importante
>
> Em caso de dúvidas sobre quais tópicos ativar ou quais eventos serão notificados, consulte a documentação [Informações adicionais sobre notificações](/developers/pt/docs/your-integrations/notifications/additional-info). 

4. Por fim, clique em **Salvar**. 

## Configuração durante a criação de pagamentos

Durante o processo de criação de pagamentos, é possível configurar a URL de notificação de maneira específica para cada pagamento. Para isso, utilize o campo `notification_url` e implemente o receptor de notificações necessário.  

> WARNING
>
> Importante
>
> Este método não permite configurar notificações para o tópico `point_integration_ipn`. Para ativá-lo, utilize as [configurações através de Suas integrações](/developers/pt/docs/your-integrations/notifications/ipn#configuraoviasuasintegraes).
 
A seguir, explicamos como realizar esta configuração utilizando nossoss SDKs.

> 1. No campo `notificaction_url`, indique a URL do qual serão recebidas as notificações como exemplificado abaixo. Para receber notificações exclusivamente via IPN e não via Webhooks, adicione o parâmetro `source_news=webhooks` à `notification_url`. Por exemplo: "https://www.yourserver.com/notifications?source_news=webhooks".

Reviewer 2: Repito comentario sobre el link cliqueable (re pesada pero es para que no lo pierdan nada más)

# IPN

IPN (Instant Payment Notification) é um mecanismo que permite que uma aplicação receba notificações do Mercado Pago sobre o status de um pagamento, _chargebacks_ e `merchant_orders`. Essas notificações são enviadas através de uma chamada HTTP POST, informando sobre as transações.

> WARNING
>
> Importante
>
> As notificações IPN serão descontinuadas. Além disso, apesar de receber o _header_ `x-Signature`, elas não permitem a validação por meio da chave secreta para confirmar que foram enviadas pelo Mercado Pago. Para realizar essa validação de origem, recomendamos migrar para as [notificações Webhooks](/developers/pt/docs/your-integrations/notifications/webhooks).   

As notificações IPN podem ser configuradas de duas maneiras: 

| Modo de configuração | Descrição |
|---|---|
| [Configuração através de Suas integrações](/developers/pt/docs/your-integrations/notifications/ipn#configuraoviasuasintegraes) | Poderá ser configurada apenas **uma URL de notificação** por conta (dependendo da aplicação, mais de uma aplicação pode usar essa URL). |

> | [Configuração durante a criação de pagamentos](/developers/pt/docs/your-integrations/notifications/ipn#configuraoduranteacriaodepagamentos) | Isso pode ser realizado a partir do campo `notification_url`. A URL poderá ser diferente para cada objeto ou aplicação.  |

Author: ```suggestion
| [Configuração durante a criação de pagamentos, preferências ou ordens comerciais](/developers/pt/docs/your-integrations/notifications/ipn#configuraoduranteacriaodepagamentos) | Isso pode ser realizado a partir do campo `notification_url`. A URL poderá ser diferente para cada objeto.  |
```

Nesta documentação explicaremos as configurações necessárias para o recebimento das notificações IPN, além de apresentar quais são as ações necessárias para que o Mercado Pago valide o correto recebimento das mensagens.

## Configuração via Suas integrações

Configure notificações diretamente via Suas integrações de forma eficiente e segura. 

### Indicar URLs e configurar eventos

Para configurar as notificações IPN via Suas integrações, é necessário indicar as URLs onde elas serão recebidas e especificar os eventos para os quais deseja receber notificações. 

> WARNING
>
> Importante
>
> Ao configurar as notificações IPN via Suas integrações, você está configurando a URL e os Eventos de **todos os aplicativos da sua conta do Mercado Pago**. 

Para configurar URLS e eventos, siga as etapas descritas abaixo:

> 1. Acesse [Suas integrações](/developers/panel/app) e selecione a aplicação para a qual deseja ativar as notificações. Caso ainda não tenha criado uma aplicação, acesse a [documentação Painel do Desenvolvedor](/developers/pt/docs/your-integrations/dashboard) e siga as instruções para fazê-lo.

Author: ```suggestion
1. Acesse [Suas integrações](/developers/panel/app) e selecione uma das aplicações para configurar as notificações para toda a sua conta. Caso ainda não tenha criado uma aplicação, acesse a [documentação Painel do Desenvolvedor](/developers/pt/docs/your-integrations/dashboard) e siga as instruções para fazê-lo.
```

2. No menu à esquerda, clique em **IPN** e configure a **URL de produção** que será usada para receber as notificações. Leve em consideração que é possível testar se a URL indicada está recebendo as notificações corretamente. Isso permite verificar a solicitação, a resposta do servidor e a descrição do evento.

![ipn](/images/dashboard/ipn_pt_.png)

> NOTE
>
> Nota
>
> Caso seja necessário identificar múltiplas contas, adicione o parâmetro `?cliente=(nomedovendedor)` ao final da URL indicada para identificar os vendedores.

> 3. Selecione os **eventos** para os quais deseja receber notificações em formato `json` através de um `HTTP POST` para a URL especificada anteriormente. Um evento pode ser qualquer atualização no objeto relatado, incluindo alterações de status ou atributos. Consulte a tabela abaixo para ver os eventos configuráveis, considerando a solução do Mercado Pago integrada e suas necessidades de negócio.

Reviewer: ```suggestion
3. Selecione os **eventos** para os quais deseja receber notificações em formato `JSON` através de um `HTTP POST` para a URL especificada anteriormente. Um evento pode ser qualquer atualização no objeto relatado, incluindo alterações de status ou atributos. Consulte a tabela abaixo para ver os eventos configuráveis, considerando a solução do Mercado Pago integrada e suas necessidades de negócio.
```

----[mla, mlm, mlb]----| Finalização, cancelamento ou erro ao processar tentativas de pagamento nos dispositivos Mercado Pago Point. | Integrações Point | `point_integration_ipn` | Mercado Pago Point |------------
| Alertas de fraude após o processamento de um pedido | Alertas de fraude | `delivery_cancellation` | Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------<br>Checkout PRO |
| Criação, fechamento ou expiração de pedidos. |  Ordens comerciais | `merchant_order` | Checkout Pro<br>Código QR  |
| Abertura de estornos, alterações de status e modificações relacionadas às liberações de dinheiro.   |   Estornos | `chargebacks`  | Checkout Pro<br>Checkout ----[mlb]----Transparente ----------------[mla, mlu, mlc, mlm, mco, mpe]----API------------ <br>Checkout Bricks |

> WARNING
>
> Importante
>
> Em caso de dúvidas sobre quais tópicos ativar ou quais eventos serão notificados, consulte a documentação [Informações adicionais sobre notificações](/developers/pt/docs/your-integrations/notifications/additional-info). 

4. Por fim, clique em **Salvar**. 

## Configuração durante a criação de pagamentos

> Durante o processo de criação de pagamentos, é possível configurar a URL de notificação de maneira específica para cada pagamento. Para isso, utilize o campo `notification_url` e implemente o receptor de notificações necessário.  

Author: ```suggestion
Durante o processo de criação de pagamentos, preferências ou ordens comerciais, é possível configurar a URL de notificação de maneira específica para cada pagamento. Para isso, utilize o campo `notification_url` e implemente o receptor de notificações necessário.  
```

----[mla, mlb, mlm]----
> WARNING
>
> Importante
>
> Este método não permite configurar notificações para o tópico `point_integration_ipn`. Para ativá-lo, utilize as [configurações através de Suas integrações](/developers/pt/docs/your-integrations/notifications/ipn#configuraoviasuasintegraes).

------------
 
A seguir, explicamos como realizar esta configuração utilizando nossoss SDKs.

> 1. No campo `notificaction_url`, indique a URL do qual serão recebidas as notificações como exemplificado abaixo. Para receber notificações exclusivamente via IPN e não via Webhooks, adicione o parâmetro `source_news=webhooks` à `notification_url`. Por exemplo: `https://www.yourserver.com/notifications?source_news=webhooks`.

Author: ```suggestion
1. No campo `notificaction_url`, indique a URL do qual serão recebidas as notificações como exemplificado abaixo. Para receber notificações exclusivamente via IPN e não via Webhooks, adicione o parâmetro `source_news=ipn` à `notification_url`. Por exemplo: `https://www.yourserver.com/notifications?source_news=ipn`.
```
